plotly/Make_a_FigureFactory.md

```diff
@@ -0,0 +1,143 @@
+# Add A Figure Factory to the Plotly [Python Library](https://plot.ly/python/)
+
+If you have ever wanted to contribute to the Plotly Python Library by adding a new chart type we don't have, now you can! This README will help you get started cloning the plotly.py repo, forking a new branch, creating a new figure factory, and creatng a new Pull Request to get feedback for merging. Just follow all these steps and you'll be ready to go.
```

Comment:
changes looks good! I'm wondering if it would be better to start out describing/clarifying the differences between figure factories (wrappers around plotly.js charts to create a new chart type) and plotly.graph_objs?